pkg/tsdb/prometheus/buffered/types.go

```diff
@@ -0,0 +1,49 @@
+package buffered
+
+import (
+	"time"
+
+	apiv1 "github.com/prometheus/client_golang/api/prometheus/v1"
+)
+
+type clientGetter func(map[string]string) (apiv1.API, error)
+
+type PrometheusQuery struct {
```

Comment:
how does this relate to:
pkg/tsdb/prometheus/types.go

Reply:
good catch. i missed removing `pkg/tsdb/prometheus/types.go`